x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1709,6 +1779,10 @@ public void clearCredentials() {
         public ApiKey.Type getExpectedType() {
             return expectedType;
         }
+
+        public String getCertificateIdentity() {
```

Comment:
Can we javadoc this?

```suggestion
        /**
         * The identity (Subject DistinguishedName) of the X.509 certificate that was provided by the client 
         * alongside the API during authenticate. 
         * <em>At the time of writing, the only place where this is used is for cross cluster request signing</em>     
         */
        public String getCertificateIdentity() {
```